totem/circles/mobile_api/mobile_api.py

```diff
@@ -136,32 +120,38 @@ def get_spaces_summary(request: HttpRequest):
         CircleEvent.objects.annotate(end_time=end_time_expression)
         .filter(attendees=user, cancelled=False, end_time__gt=timezone.now())
         .select_related("circle")
-        .prefetch_related("circle__author", "circle__categories", "attendees")
-        .annotate(attendee_count=Count("attendees", distinct=True))
+        .prefetch_related("circle__author", "circle__categories", "attendees", "circle__subscribed")
+        .annotate(
+            attendee_count=Count("attendees", distinct=True),
+            subscriber_count=Count("circle__subscribed", distinct=True),
+        )
         .order_by("start")
     )
     upcoming = [event_detail_schema(event, user) for event in upcoming_events]
+    upcoming_circle_slugs = {event.circle.slug for event in upcoming_events}
 
     # The recommended spaces based on the user's onboarding.
-    onboard_model = get_object_or_404(OnboardModel, user=user)
     categories_set = set()
-    if onboard_model.hopes:
-        for hope in onboard_model.hopes.split(","):
-            name = hope.strip()
-            if name:
-                categories_set.add(name)
+    try:
+        onboard_model = OnboardModel.objects.get(user=user)
+        if onboard_model.hopes:
+            for hope in onboard_model.hopes.split(","):
+                name = hope.strip()
+                if name:
+                    categories_set.add(name)
+    except OnboardModel.DoesNotExist:
+        # If no onboard model, just use empty categories set
+        pass
```

Comment:
Would be fun if this could work, but people usually put complete sentences in this field. It's unlikely to catch any interesting categories, just an extra db call.